Sparql. Illia Ovtsynov.txt

```diff
@@ -0,0 +1,70 @@
+QUESTION 1
+What license does the <http://resources.samsung.com/projects/.net> project have?
+
+SPARQL QUERY 1
+LICENSE:
+MIT
+
+QUESTION 2
+Which repositories does the <http://resources.samsung.com/projects/.net> project have?
+
+SPARQL QUERY 2
+REPOSITORIES:
+http://resources.samsung.com/repositories/dotnet/runtime
+
+QUESTION 3
+Which license and description does the <http://resources.samsung.com/projects/.net> project have?
+
+SPARQL QUERY 3
+LICENSE:
+MIT
+DESCRIPTION:
+
+
+QUESTION 4
+Who contributed to the <http://resources.samsung.com/repositories/boostorg/boost> repository?
+
+SPARQL QUERY 4
+CONTRIBUTORS:
+http://resources.samsung.com/contributors/ericniebler
```

Comment:
To return only login, you can add triple, that contain login info. (and return only this login)